### PR TITLE
Limit groundspeed in UI to 3m/s

### PR DESCRIPTION
**Pull request by haslinghuis**
Requested by @ctzsnooze 

Limit's groundspeed to 3.0 m/s instead of 0.3 m/s
CLI can override this range. As long user does not alter the setting in UI the CLI range is used otherwise the UI range comes into effect.

### CODE REVIEW

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Thanks, @haslinghuis, this will keep the ordinary user's groundspeed above the minimum necessary for correct IMU heading calculations.
An advanced user can use the CLI to set a lower value, including zero, for specialised testing purposes.
3.0m/s is probably the minimum safe value for a reliable GPS Rescue.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
